plugins/outputs/dcos_metrics/translator.go

```diff
@@ -50,6 +50,10 @@ func (t *producerTranslator) Translate(metric telegraf.Metric) (msg producers.Me
 	case hasAllKeys(tags, []string{"container_id", "metric_type"}):
 		msg = t.appMetricsMessage(metric)
 
+	// We assume any metric with both a container_id tag and a url tag is an app metric from the prometheus input.
+	case hasAllKeys(tags, []string{"container_id", "url"}):
+		msg = t.appMetricsMessage(metric)
```

Comment:
I think it might be clearer to express the app metrics case with a single `case`:

```go
case hasAllKeys(tags, []string{"container_id"}) && hasAnyKeys(tags, []string{"metric_type", "url"})
```

But I don't have a strong enough preference to block the PR on it, especially given that this function is well covered by tests.